### ISSUE TITLE
Issue when deleting CAD character license plates

### ISSUE THREAD

**Issue by ladojrp-sourdirt** [ScottUK/ladojrp-issues#84]
**Description of the problem**
When you delete a character from the CAD system I've noticed any vehicles/license plates stay locked to that character and you can't re-register those license plates to a new character.

**Reproduction steps**
Go into the CAD, create a character and then register a vehicle/license plate. Next delete the character without removing the license late/car registration. Now if you go into a different character that you already have or create a new one you can no longer use that previously registered vehicle/license plate.

**Expected behavior**
I was hoping once you delete a character from CAD all previously registered vehicle license plates would be able to be registered under a new/existed character.

**Screenshots**
N/A

**Additional information**
Basically I had 3 vehicle license plates that were registered previously on a character that I had deleted and now I can't register them under an existing or new character in the CAD and with that issue nobody will be able to think of good plates and/or reuse them if they were to get "transfered" to a different person or vehicle especially RP-wise.

**Comment by LadojrpAustin** [comment]
Hello SourDirt, I am Austin with the LADOJRP Suggestions Team

Thank you for this suggestion and it will be looked at very carefully and it will be taken into consideration

Since this is a bug and I won't be able to do much about it I will just pass this up and this will be looked at and fixed 

Thank you for the suggestion and have a great day!

Administrator, Austin

**Comment by LadojrpAustin** [comment]
@ladojrp-alopez

**Comment by LADOJRP-Mello** [comment]
@ScottUK
